spikeinterface/core/core_tools.py

```diff
@@ -11,6 +12,7 @@
 
 from .job_tools import (ensure_chunk_size, ensure_n_jobs, divide_segment_into_chunks, fix_job_kwargs, 
                         ChunkRecordingExecutor, _shared_job_kwargs_doc)
+import spikeinterface.core.base as base
```

Comment:
When submodule partial import I thin I prefer this 
```suggestion
from spikeinterface.core.base import BaseExtractor
```